uaclient/config.py

```diff
@@ -285,12 +287,12 @@ def _attached_status(self) -> "Dict[str, Any]":
                 contractInfo["effectiveTo"], "%Y-%m-%dT%H:%M:%SZ"
             )
 
-        resources = get_available_resources(self)
-        inapplicable_resources = [
-            resource["name"]
+        resources = self.machine_token.get("availableResources")
```

Comment:
Does this also fix #760?

Reply:
This does fix #760 too... but we will likely have to sort this feature on the contract server as a result of this shift. https://github.com/CanonicalLtd/ua-contracts/issues/499

Reply:
We could potentially switch things so that we display the custom message iff we determine based on the local system that an entitlement is inapplicable? That way we don't need to update the contract server's state.

The full solution for that ua-contracts issue would drive non-trivial client changes, I think? That's the only reason I'm wondering if there are more minimal changes we can make in the short-term to get this landed and usable in trusty.

Reply:
Ok, I've pushed an update but still needs unit tests for both paths. We can work around this shortcoming, but for ua-contracts #499 the refreshed token doesn't even contain availableResources at all so we'll have to fallback to get_avail_resources until 499 gets a fix. And you are right that ua-contracts change is potentially not trivial to uaclient as we'll have to pass more form data to the route